pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/OffloadPolicies.java

```diff
@@ -41,7 +41,7 @@
     public final static String[] DRIVER_NAMES = {"S3", "aws-s3", "google-cloud-storage", "filesystem"};
     public final static String DEFAULT_OFFLOADER_DIRECTORY = "./offloaders";
     public final static long DEFAULT_OFFLOAD_THRESHOLD_IN_BYTES = -1;
-    public final static Long DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS = null;
+    public final static long DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS = -1;
```

Comment:
Should we also change `managedLedgerOffloadDeletionLagInMillis` to `long`?

Reply:
This default is couldn't be `Null`, so I changed this field type to long.

Reply:
Yes, it's fine. I was just saying to also change the type for the `managedLedgerOffloadDeletionLagInMillis` field which is currently `Long`. If `null` is not a valid value, we should use `long` type instead.

Reply:
Yes, I changed it. Thanks.